.circleci/deploy_to_kubernetes.sh

```diff
@@ -56,6 +56,33 @@ deploy(){
 		terraform apply -target module.k8s -auto-approve
 		# update the state
 		gsutil cp terraform.tfstate gs://$STAGING_BUCKET/$STAGING_PREFIX/default.tfstate
+	elif [ "$CIRCLE_BRANCH" == qa ]; then
+	# set credentials
+		echo $GOOGLE_CREDENTIALS_SANDBOX | base64 --decode > secrets/google-creds-sandbox.json
```

Comment:
Nice work @mariamiah, just something to point out is that I don't think we should deal with the sandbox when the branch is QA.